### DISCUSSION TITLE
dotnet tool install gherkin fails

### DISCUSSION THREAD

**Discussion by TesterEric** [community/community#143952] [Programming Help]
### Select Topic Area

Question

### Body

`dotnet tool install gherkin` fails (after running `dotnet new tool-manifest`)

![image](https://github.com/user-attachments/assets/3605d802-3f86-4fea-9b3f-a49c24391e7f)

What could be the reason for this? Is gherkin available as CLI tool (https://github.com/cucumber/gherkin?tab=readme-ov-file#readme mentions it is)?  Other tools can be installed successfully without any error.

**Comment by urmasa-tar** [comment]
Hi
An error with the message "Could not find a part of the path" usually indicates that the application cannot find the specified file or folder. Here are a few steps to diagnose and solve this problem:

1. Checking The Path
Make sure that the path specified in your code actually exists. If you are using relative paths, try using absolute paths to make sure you are pointing to the correct location.

2. Checking The Register Sensitivity
If you are working on a system where case-insensitivity of the path is important (for example, on Linux), make sure that the letter case matches the actual names of files and folders.

3. Access Rights
Make sure that your application has the necessary read/write access rights to the specified directories or files. If you are working in an environment where special permissions are required, try running the application with elevated permissions.

4. Error in the File or Folder Name
Check if there are any typos in the file or folder name. Make sure that all special characters and file extensions are specified correctly.

5. Problems with Dependencies
If your code depends on any files that need to be created or copied before execution, make sure that all dependencies are in the expected location.

6. Operating System
Please note that file paths may vary between different operating systems. If your code is developed on Windows and runs on Linux (or vice versa), make sure that the paths are specified correctly for the target platform.

7. Debugging
Add debugging messages or use the debugger to see exactly which path is being generated at the time of the call. This will help clarify exactly where the error is occurring.

**I'm not an english native speeker and that's why part there are may be some errors in my text. I hope that will be cleare and undestandable anothe.**
**Good luck and have a good  day!**

**Comment by TesterEric** [comment]
This does not solve the issue. I am able to install other dotnet tools `C:\Users\<user>\.nuget\packages\`, but somehow gherkin cannot be installed. If I add the folder `tools` to this nuget folder the command fails with: _Unhandled exception: Microsoft.DotNet.ToolPackage.ToolConfigurationException: Settings file 'DotnetToolSettings.xml' was not found in the package._
